src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -257,6 +257,30 @@ def test_WINNF_FT_S_SCS_15(self,config_filename):
     # Check registration response
     self.assertEqual(response['response']['responseCode'],104)
 
+  def generate_SCS_16_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_16. """
+    # Create the configuration for client cert/key path.
+
+    config = {
+      'clientCert': self.getCertFilename("client.cert"),
```

Comment:
Is this cert derived from a revoked sub-CA?

Reply:
No. Now   we have created the new client certificate signed by revoked sub CA.